src/main/java/de/hysky/skyblocker/utils/render/SkyblockerRenderPipelines.java

```diff
@@ -44,6 +44,29 @@ public class SkyblockerRenderPipelines {
 			.withCull(false)
 			.build());
 
+	static final RenderPipeline SPHERE = RenderPipelines.register(RenderPipeline.builder(RenderPipelines.POSITION_COLOR_SNIPPET)
+			.withLocation(Identifier.of(SkyblockerMod.NAMESPACE, "pipeline/sphere"))
+			.withVertexFormat(VertexFormats.POSITION_COLOR, DrawMode.TRIANGLE_STRIP)
+			.withCull(false)
+			.build());
+
+
+
+	static final RenderPipeline CIRCLE = RenderPipelines.register(RenderPipeline.builder(RenderPipelines.POSITION_COLOR_SNIPPET)
+			.withLocation(Identifier.of(SkyblockerMod.NAMESPACE, "pipeline/circle"))
+			.withVertexFormat(VertexFormats.POSITION_COLOR, DrawMode.TRIANGLE_FAN)
+			.withCull(false)
+			.build());
+	static final RenderPipeline CIRCLE_LINES = RenderPipelines.register(RenderPipeline.builder(RenderPipelines.POSITION_COLOR_SNIPPET)
+			.withLocation(Identifier.of(SkyblockerMod.NAMESPACE, "pipeline/circle_lines"))
+			.withVertexFormat(VertexFormats.POSITION_COLOR, DrawMode.QUADS)
+			.withCull(false)
+			.build());
+
+
+
+
+
```

Comment:
The formatting is a bit messed up. We usually do zero empty lines between fields like this. Also for the file above.